osgp/platform/osgp-domain-distributionautomation/src/main/java/org/opensmartgridplatform/domain/da/measurements/Measurement.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.domain.da.measurements;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class Measurement {
+    private List<MeasurementElement> measurementElements = new ArrayList<>();
+
+    public static Measurement of(final MeasurementElement... elements) {
+        return new Measurement(Arrays.asList(elements));
+    }
+
+    public Measurement(final List<MeasurementElement> measurementElements) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1213)